tests/beaker_tests/radius_server/radius_server_provider_defaults.rb

```diff
@@ -89,13 +89,13 @@
       search_pattern_in_output(stdout, { 'ensure' => 'present' },
                                false, self, logger)
       search_pattern_in_output(stdout, { 'accounting_only' => 'true' },
-                               false, self, logger)
+                               false, self, logger) unless platform == 'xrv9k'
```

Comment:
Better to take a queue from operatingsystem as xrv9k is not the only IOS XR platform.